doc/rose-rug-advanced-tutorials-arch.html

```diff
@@ -0,0 +1,411 @@
+    </div>
+
+    <div class="header-footer" id="footer"></div>
+  </div>
+
+  <div id="body-main" class="presentation">
+    <div class="slide">
+      <h1>Rose Advanced Tutorial: rose_arch</h1>
+    </div>
+
+    <div class="handout" id="content"></div>
+
+    <div class="slide">
+      <h2 id="intro">Introduction</h2>
+
+      <p>This tutorial walks you through developing <a href=
+      "rose-api.html#macro">rose_arch</a> app's - built-in application
+      that provides a generic solution to configure site specific archiving
+      of suite files.</p>
+
+      <p>This allows users to define which files to archive and to reduce disk
+      space used for archived files which keeping data for prosperity.</p>
+    </div>
+
+    <div class="slide">
+      <h2 id="warning">Warning</h2>
+
+      <p>Only archive the minimum files needed at each cycle of your suite.
+      Run this before your <code>housekeep</code> app in the graphing of your
+      suite.rc so that you do not lose valuable data that is needed.</p>
+
+    </div>
+
+    <div class="slide">
+      <h2 id="purpose">Purpose</h2>
+
+      <p><code>archive</code> app's are user-defined app's that use rose_arch
+      , the built-in application to archive data from a defined source to a
+      defined location using a defined command. The follow examples will
+      provide a guide to using rose_arch with the user-defined archive
+      app.</p>
+    </div>
+
+    <div class="slide">
+      <h2 id="syntax">Syntax</h2>
+
+      <p>The regular expression syntax is Pythonic. Some aspects use regular 
+      expressions
+      <a href="https://docs.python.org/2/howto/regex.html#">PythonHOWTO</a>
+      how these are used in the app/archive/ is explained in detail later.</p>
+    </div>
+
+    <div class="slide">
+      <h2 id="example">Example</h2>
+
+      <p>Create a <a href="rose-rug-brief-tour#cli">new suite</a> (or just a
+      new directory somewhere - e.g. in your homespace) containing a blank
+      <samp>rose-suite.conf</samp> and a <samp>suite.rc</samp> file that looks
+      like this:</p>
+    </div>
+
+    <div class="slide">
+    <h3 class="alwayshidden">Example suite.rc</h3>
+    <pre class="prettyprint lang-cylc">
+[cylc]
+    UTC mode = True # Ignore DST
+    [[event hooks]]
+        timeout handler = rose suite-hook --mail --shutdown
+        timeout = PT1H
+[scheduling]
+    [[dependencies]]
+        graph = create_files =&gt; archive_files_rsync =&gt; archive_files_scp
+[runtime]
+    [[root]]
+        environment scripting = eval $(rose task-env)
+        command scripting = rose task-run
+        [[[event hooks]]]
+            succeeded handler = rose suite-hook
+            failed handler = rose suite-hook
+</pre>
+    </div>
+
+    <div class="slide">
+       <h3 class="alwayshidden">Example suite.rc (2)</h3>
+       <pre class="prettyprint lang-cylc">
+    [[create_files]]
+        command scripting = """
+            echo 'zip' &gt;&gt; $ROSE_DATAC/file_zip
```

Comment:
It assigns it a value of 1 as `ROSE_DATAC` for the non-cycling suite and this can be used. I wanted to show the `ROSE_DATAC` to enhance the tutorial for users to see that they can reference the current cycle. I am happy to remove it if you wish.